.gitignore

```diff
@@ -81,6 +81,12 @@ docs/_build/
 # PyBuilder
 target/
 
+# thirdparty dirs
+third_party/*/
+
+# computed checksum files
+torchtext/experimental/asset/.checksums/
```

Comment:
Do we still have those files?

Reply:
Yup so these get generated by the script I checked in to compute checksums. The checksum dictionary needs to be manually copied from the generated JSON file and pasted into the `vectors.py` file after running the script